pkg/skel/skel.go

```
@@ -93,14 +93,20 @@ func PluginMain(cmdAdd, cmdDel func(_ *CmdArgs) error) {
 	}
 
 	if err != nil {
+		if e, ok := err.(*plugin.Error); ok {
+			// don't wrap Error in Error
+			e.Print()
```

this is swallowing the error from Print?